src/components/dashboard/FeedItems/EventSettingsByType.js

```diff
@@ -61,6 +61,21 @@ const getEventSettingsByType = (theme, type) => {
       color: theme.colors.green,
       name: 'receive-filled',
     },
+    withdrawerror: {
+      actionSymbol: '+',
```

Comment:
ask liav about the colors and icons

Reply:
asked Liav. Left comment in task description and fix